docs/subsystems/html-css.md

```diff
@@ -38,6 +44,10 @@ understand the current styling or modify it. We would very much like
 to avoid such a fate. So please make an effort to reuse existing
 styling, clean up now-unused CSS, etc., to keep things maintainable.
 
+Opt to write CSS in CSS files. Avoid using the `style=` attribute in
+HTML unless the styling is actually dynamic. Instead, define logical
```

Comment:
"actually dynamic" might benefit from an example, like the stream colors that depend on user data.